### ISSUE TITLE
Support Rate and Voice Settings on Windows

### ISSUE THREAD

**Issue by mattbierner** [Marak/say.js#46]
The voice and rate options currently do not seem to have an effect on Windows. If change #45 is taken in, this is easy enough to add using `SelectVoice` and `Rate` in the powershell script:

``` ps
Add-Type -AssemblyName System.speech;
$speak = New-Object System.Speech.Synthesis.SpeechSynthesizer;
$speak.SelectVoice("Microsoft Zira Desktop");
$speak.Rate = 3;
$speak.Speak([Console]::In.ReadToEnd());
```

I can submit a PR if #45 gets merged

**Comment by tlhunter** [comment]
#45 did get merged if you want to take a crack at it

**Comment by JimmyBoh** [comment]
I might be able to look into this.

**Comment by JimmyBoh** [comment]
For anyone who does have windows and some spare time, please take a look at my pull request. Windows accepts speeds `[-10,+10]` so I created a mapping equation based on my studies. Let me know if anyone knows a better way or if they get different results (I can check in my test file if anyone is interested)!

**Comment by tlhunter** [comment]
Fixed in #67

**Comment by tlhunter** [comment]
Published in **v0.14.0**.
